string mode
Local local
int16 index
float64[] path_x
float64[] path_y
float64[] path_heading
float64[] path_local_x
float64[] path_local_y

geometry_msgs/Point32 point
